js/controller.js

```diff
@@ -192,6 +192,18 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     onAuthorizationFetched: function(event, authorization) {
       this.state.authorization = authorization;
+
+      this.mb.subscribe(OO.EVENTS.VC_PLAYED, 'customerUi', _.bind(this.onVcPlayed, this));
```

Comment:
@aldigun Have added a state check before re-subbing the events. However the "OO.EVENTS.PLAYBACK_READY" has to be present in order to perform the state check, so it was left outside the if condition.